.github/workflows/filestore.yaml

```
@@ -51,3 +50,16 @@ jobs:
           docker build  -t selcohub/egov-filestore-db:${{ env.VERSION }}-${{ env.commit_hash }} -f  Dockerfile .
           docker push selcohub/egov-filestore-db:${{ env.VERSION }}-${{ env.commit_hash }}
         working-directory: backend/core-services/egov-filestore/src/main/resources/db
+
+Workflow_Trigger:
```

actionlint (1.7.4)

unexpected key "Workflow_Trigger" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)